azure-pipelines.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         displayName: yarn
```

Comment:
Can we move out these changes into a separate PR?

Reply:
sure, it will just block this PR 😞

Reply:
Done in #24395

Reply:
Reverted yml changes in https://github.com/microsoft/fluentui/pull/24386/commits/00d3f72ac182d384869995bd0550f1f2677cc7b6